Detectors/FIT/FT0/calibration/testWorkflow/LHCphaseCalibrationSpec.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_CALIBRATION_LHCCLOCK_CALIBRATOR_H
+#define O2_CALIBRATION_LHCCLOCK_CALIBRATOR_H
```

Comment:
This repeats the guards of Detectors/TOF/calibration/testWorkflow/LHCClockCalibratorSpec.h
The same I see in the Detectors/FIT/FT0/calibration/testWorkflow/GlobalOffsetsCalibrationSpec.h
Could you please change both to FT0-header specific?

Reply:
Done